meta-lmp-base/classes/flashlayout-stm32mp-archiver.bbclass

```diff
@@ -0,0 +1,16 @@
+do_create_flashlayout_tarball() {
+    tar -chzf ${DEPLOY_DIR_IMAGE}/flashlayouts-${MACHINE}.tar.gz \
+        -C ${DEPLOY_DIR_IMAGE} arm-trusted-firmware/ \
+        fip/ \
+        ${MFGTOOL_FLASH_IMAGE}.bin flashlayout_${IMAGE_BASENAME}/
+}
+
+python __anonymous () {
+    machine = d.getVar("MACHINE", True)
+    if machine.find('stm32mp1') != -1:
+        tasks = filter(lambda k: d.getVarFlag(k, "task", True), d.keys())
+        for task in tasks:
```

Comment:
Can't we use if 'do_create_flashlayout_config' in tasks instead?

Reply:
yes, will fix